packages/cli/jest-puppeteer.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  launch: {
+    headless: true, // set to false to view browser window
+  },
+  server: {
```

Comment:
I just checked the APIs for jest-puppeteer, which does allow launching of multiple servers of different setup [here](https://github.com/smooth-code/jest-puppeteer/tree/master/packages/jest-dev-server#usage)
> ![image](https://user-images.githubusercontent.com/41845017/156150853-d3dfbe58-72f8-4be7-b983-608800213687.png)

If this works then I think I will need to have two different sets of test sites to prevent side effects. So after tests on -f is done, proceed to test with -o which is serving content from another site. Do you think this is fine?

Reply:
Ideally: setup -> teardown (also cleanup site edits, so we can keep to 1 test site perhaps) -> setup -> teardown, as there are quite a few combinations which might overload gh-actions. (e.g. `-o` `-o -b` `-b` `-f` ...)